inference-engine/tests_deprecated/functional/vpu/common/layers/myriad_layers_resample_test.hpp

```diff
@@ -61,12 +63,21 @@ void refResample(const Blob::Ptr src, Blob::Ptr dst, int antialias) {
         {
             for (int ox = 0; ox < OW; ox++)
             {
-                float ix = ox*fx + fx / 2.0f - 0.5f;
+                float ix = ox*fx + fx / 2.0f - 0.5f; // half pixel
                 float iy = oy*fy + fy / 2.0f - 0.5f;
 
-                int ix_r = (int)(round(ix));
+                int ix_r = (int)(round(ix)); // round prefer ceil
                 int iy_r = (int)(round(iy));
 
+                if (coordTransMode == 2) { // asymmetric
```

Comment:
Is it any chance that we can reuse `InterpolateCoordTransMode` here to avoid using `int`?

Reply:
I guess, yes. Will do.

Reply:
done.